src/group_theory/quotient_group.lean

```diff
@@ -266,6 +266,29 @@ def equiv_quotient_of_eq {M N : subgroup G} [M.normal] [N.normal] (h : M = N) :
   right_inv := λ x, x.induction_on' $ by { intro, refl },
   map_mul' := λ x y, by rw map_mul }
 
+/-- Let `A', A, B', B` be subgroups of `G`. If `A' ≤ B'` and `A ≤ B`,
+then there is a map `A / (A' ⊓ A) →* B / (B' ⊓ B)` induced by the inclusions. -/
+@[to_additive "Let `A', A, B', B` be subgroups of `G`. If `A' ≤ B'` and `A ≤ B`,
+then there is a map `A / (A' ⊓ A) →+ B / (B' ⊓ B)` induced by the inclusions."]
+def quotient_map_of_subgroups_le {A' A B' B : subgroup G}
+  [hAN : (A'.subgroup_of A).normal] [hBN : (B'.subgroup_of B).normal]
+  (h' : A' ≤ B') (h : A ≤ B) :
+  quotient (A'.subgroup_of A) →* quotient (B'.subgroup_of B) :=
+map _ _ (subgroup.inclusion h) $
+  by simp [subgroup.subgroup_of, subgroup.comap_comap]; exact subgroup.comap_mono h'
+
+/-- Let `A', A, B', B` be subgroups of `G`.
+If `A' = B'` and `A = B`, then the quotients `A / (A' ⊓ A)` and `B / (B' ⊓ B)` are isomorphic. -/
+@[to_additive "Let `A', A, B', B` be subgroups of `G`.
+If `A' = B'` and `A = B`, then the quotients `A / (A' ⊓ A)` and `B / (B' ⊓ B)` are isomorphic."]
+def equiv_quotient_of_subgroups_eq {A' A B' B : subgroup G}
```

Comment:
I think this name should include `subgroup_of` in it since it is about `subgroup_of` not `of_subgroups`. So maybe
```suggestion
def equiv_quotient_subgroup_of_of_eq {A' A B' B : subgroup G}
```